cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -430,19 +425,14 @@ func (fns *containerAppTemplateManifestFuncs) ConnectionString(name string) (str
 		}
 
 		if parent.Type == "container.v0" {
-			rawConnectionString := strings.Split(strings.TrimRight(*parent.ConnectionString, ";"), ";")
-			rawConnectionString = append(rawConnectionString, fmt.Sprintf("Database=%s;", name))
-			for i, part := range rawConnectionString {
-				keyValue := strings.Split(part, "=")
-				resolvedValue, err := apphost.EvalString(keyValue[1], func(expr string) (string, error) {
-					return evalBindingRefWithParent(expr, parent, fns.env)
-				})
-				if err != nil {
-					return "", fmt.Errorf("evaluating connection string for %s: %w", name, err)
-				}
-				rawConnectionString[i] = fmt.Sprintf("%s=%s", keyValue[0], resolvedValue)
+			rawConnectionString := *parent.ConnectionString + fmt.Sprintf("Database=%s;", name)
```

Comment:
FWIW, might still be worth to ensure there's a trailing `;` for robustness

Reply:
done